classes/Link.php

```diff
@@ -1587,4 +1592,34 @@ public static function getUrlSmarty($params)
 
         return $link;
     }
+
+    /**
+     * @param Shop $shop
+     * @param int|null $idShop
+     *
+     * @return array
```

Comment:
```suggestion
     * @return array<string>
```

Reply:
Hello @Progi1984 
I have updated the comment in the following commit.